test/integration/strong_mode_test.dart

```diff
@@ -0,0 +1,89 @@
+@TestOn('vm')
+library over_react.strong_mode;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/util.dart' show copyDirectory;
+import 'package:test/test.dart';
+
+const String strongModeProject = 'test_fixtures/strong_mode';
+
+/// Runs the dart analzyer task for the given project.
+Future<ProcessResult> analyzeProject(String projectPath) async {
+  var files = <String>[]
+    ..addAll(new Directory(projectPath).listSync()
+        .where((e) => FileSystemEntity.isFileSync(e.path) && e.path.endsWith('.dart'))
+        .map((e) => e.path)
+    );
+
+  var args = ['--strong']
+    ..addAll(files);
+
+  return await Process.run('dartanalyzer', args);
+}
+
+// Returns the path to a new temporary copy of the [project] fixture
+// testing purposes, necessary since formatter may make changes to ill-formatted files.
+String copyProject(String project) {
+  final String testProject = '${project}_temp';
+
+  final Directory temp = new Directory(testProject);
+  copyDirectory(new Directory(project), temp);
+  addTearDown(() {
+    // Clean up the temporary test project created for this test case.
+    temp.deleteSync(recursive: true);
+  });
+
+  return testProject;
+}
+
+String getPubspec() {
+  var lines = <String>[
+    '# Generated by strong_mode_test.dart',
+    'name: test_fixture',
+    'version: 0.0.0',
+    'dependencies:',
+    ' over_react:',
+    '   path: ../..'
+  ];
+
+  return lines.join('\n');
+}
+
+main() {
+  group('OverReact strong mode compliance:', () {
+    test('generates strong mode compliant code', () async {
+      var testProject = copyProject(strongModeProject);
+
+      // Run `pub get` and the bootstrap up-front so that back-to-back runs on the same directory complete faster.
```

Comment:
I think this comment was relevant to this code being run in `setUpAll`, and is no longer applicable here.